l5-operator/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/opdev/l5-operator-demo/l5-operator
+
+go 1.16
```

Comment:
version 1.17 here as well. and latest of other dependencies as well. ie controller-runtime: 0.11.0, client-go: 0.23.0, apimachinery 0.23.0 updating that would require a `go mod tidy` as well.